lib/jekyll-gist/gist_tag.rb

```diff
@@ -4,7 +4,12 @@ class GistTag < Liquid::Tag
 
       def render(context)
         if tag_contents = determine_arguments(@markup.strip)
-          gist_id, filename = tag_contents[0], tag_contents[1]
+          gist_id = context.registers[:page]['gist_id']
```

Comment:
You may have many gists you want, so I don't think this would be a good idea necessarily. I'd rather not rely on front matter if possible.